src/components/pages/Groups/EditGroupPanel/EditGroupPanel.test.tsx

```diff
@@ -17,11 +17,17 @@ import {
   getPatchGroupsIdIdentitiesMockHandler,
   getPatchGroupsIdIdentitiesMockHandler400,
   getDeleteGroupsIdIdentitiesIdentityIdMockHandler400,
+  getPostGroupsIdRolesMockHandler,
+  getDeleteGroupsIdRolesRolesIdMockHandler,
+  getGetGroupsIdRolesMockHandler,
+  getGetGroupsIdRolesResponseMock,
+  getPostGroupsIdRolesMockHandler400,
 } from "api/groups-id/groups-id.msw";
 import { Label as EntitlementsPanelFormLabel } from "components/EntitlementsPanelForm/types";
 import { hasToast, renderComponent } from "test/utils";
 
 import { Label as IdentitiesPanelFormLabel } from "../IdentitiesPanelForm/types";
+import { Label as RolesPanelFormLabel } from "../RolesPanelForm/types";
```

Comment:
This might be a little pedantic, but I think the imports for the labels are done inconsistently e.g. `EntitlementsPanelFormLabel` is imported with the absolute path, while the others are imported with a relative path. I think this might be an issue coming from the "Copy with imports" plugin in Visual Studio Code, as it copies the import with the relative path.

Reply:
Fixed.